src/crkeng/site/settings.py

```diff
@@ -35,6 +35,8 @@
 STRICT_ANALYZER_FST_FILENAME = "crk-strict-analyzer-for-dictionary.hfstol"
 RELAXED_ANALYZER_FST_FILENAME = "crk-relaxed-analyzer-for-dictionary.hfstol"
 
+MORPHODICT_SUPPORTS_AUTO_DEFINITIONS = True
```

Comment:
My current thinking is that all the documentation comments for morphodict settings should go into `morphodict.site.settings`, instead of being duplicated in different language-specific files.  I’ve documented the option and set the default there.